code/datums/webhook.dm

```diff
@@ -0,0 +1,71 @@
+var/list/global_webhooks = list()
+
+/hook/startup/proc/initialize_webhooks()
+	return global_initialize_webhooks()
+
+proc/global_initialize_webhooks()
+	var/list/Lines = file2list("config/webhooks.txt")
+	for(var/line in Lines)
+		if(!length(line))
+			continue
+		if(copytext(line,1,2) == "#")
+			continue
+
+		var/list/Parts = text2list(line, "-")
+		if(!Parts.len && Parts < 2)
+			continue
+
+		var/url = trim(Parts[1])
+		var/list/Tags = text2list(trim(lowertext(Parts[2])), ";")
+		if(!Tags.len)
+			continue
+		var/datum/webhook/W = new(url, Tags)
+		global_webhooks += W
+	return 1
+
+/datum/webhook
+	var/url = ""
+	var/list/tags
+
+/datum/webhook/New(var/U, var/list/T)
+	. = ..()
+	url = U
+	tags = T
+
+/datum/webhook/proc/send(var/Data)
+	var/res = send_post_request(url, Data, "Content-Type: application/json")
+	switch (res)
+		if (-1)
+			return 0
+		if (0 to 90)
+			log_debug("Webhooks: cURL error while sending: [res]. Data: [Data].")
+			return 0
+		else
+			return 1
+
+/proc/post_webhook_event(var/tag, var/list/data)
+	var/OutData = list()
+	switch (tag)
+		if ("g_cciaa")
+			OutData["content"] = data["message"]
+		if ("g_admin")
+			OutData["content"] = data["message"]
+		if ("roundend")
+			var/emb = list(
+				"title" = "Round has ended",
+				"color" = 16711680,
```

Comment:
What is this value?

Reply:
By api it is decimal representation of 0xFF0000, what corresponds to rgb(255, 0, 0), a.k.a. redest red. I also forgot to show how it looks on discord.
![image](https://user-images.githubusercontent.com/11229891/32010750-5ecdd39c-b9bb-11e7-9b3e-98b85418b697.png)

Reply:
Should use a local define. Just undefine it at the end.

Reply:
Could also just put the hex in, `0xFF0000` would be perfectly valid here.